sql/src/main/java/io/crate/execution/jobs/CumulativePageBucketReceiver.java

```diff
@@ -185,21 +198,22 @@ private void triggerConsumer() {
     }
 
     private void mergeAndTriggerConsumer() {
```

Comment:
maybe we should rethink the method names as this one itself is not merging anymore (but the triggerConsumer() is now)